concrete/src/Form/Service/Widget/DateTime.php

```diff
@@ -302,12 +302,15 @@ public function date($field, $value = null, $calendarAutoStart = true)
         if ($requestValue !== null) {
             $dateTime = $requestValue;
         } elseif ($value) {
-            if (!$value instanceof PHPDateTime) {
-                try {
-                    $dateTime = $dh->toDateTime($value);
-                } catch (Exception $x) {
-                }
-            }
+            if ($value instanceof PHPDateTime) {
+              $dateTime = clone $value;
+              $dateTime->setTimezone($dh->getTimezone('user'));
+            } else {
+              try {
+                $dateTime = $dh->toDateTime($value, 'user', 'system');
```

Comment:
Ok, bottom line, the code never executed in the case of the Date function and the script always returned a blank date even when one is available.  Perhaps you could test the date picker for date only and see what I'm seeing and suggest a better fix.  Thank you for your feedback.